Sources/Testing/Events/Recorder/Event.HumanReadableOutputRecorder.swift

```diff
@@ -62,6 +62,9 @@ extension Event {
 
         /// The number of known issues recorded for the test.
         var knownIssueCount = 0
+        
+        /// The number of test cases for the test.
+        var testCasesCount: Int = 0
```

Comment:
nit: can omit the explicit type since this isn't a `public` property, to match prevailing style
```suggestion
        var testCasesCount = 0
```